### ISSUE TITLE
[Programmatic Access - Azure Databases Extension - Open Query editor(Preview)]: Visible name and accessible name are different for the "Duplicate" button.

### ISSUE THREAD

**Issue by ChandiniSmiley** [microsoft/vscode-cosmosdb#2661]
### **User Experience:** 

Screen reader-dependent users will find it difficult to understand the purpose of the controls if the visible name and Accessible name (pro-grammatically defined name) are different and will be misguided to get the right information of the control. Hence will fail to perform any action related to that particular control present on the page.  

**Note:** Login Credentials must not be provided within the bug. 

### **Pre-requisite:** 

Azure database should be installed from extensions in VS code. 

### **Repro Steps** 

1. Open VS Code and navigate to azure databases button present on right pane. 
2. Navigate to anyone of the resource and expand it. Expand a database. Right click on any collection and select "Open Query editor(Preview)". 
3. Query editor screen appears. Navigate through all controls of the screen like query editing and all the controls appearing. 
4. Verify if Visible name and accessible name are different for the "Duplicate" button. 

### **Actual Result:** 

Case 1: Visible name and accessible name are different for the "Duplicate" button i.e Accessible name is defined "Copy into new tab" and visible name is "Duplicate". 

Case 2: Visible name and accessible name are different for the "Refresh" button i.e Accessible name is defined "Reload original document from the database" and visible name is "Refresh".

### **Expected Result:** 

Case 1:

Solution: Visual name and aria-label should be same for the "Duplicate" button. 

Example: Name should be defined for switch as "Duplicate into new tab". 

Case 2:

Solution: Visual name and aria-label should be same for the "Refresh" button. 

Example: Name should be defined  as "Refresh original document from the database".

### **MAS Reference:** 

[MAS 2.5.3 – Label in Name](https://aka.ms/MAS2.5.3)

### **Reference links:** 
 
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries. 

### **Test Environment:**   

- OS Version: 24H2 (OS Build 26100.3476) 
- Visual Studio Code Version: 1.99.0 (user setup) 
- Tool: Accessibility Insights for Win32 
- Azure Databases for VS Code (Preview) Version : 0.25.2 
- Matrix:Win32+Narrator

<img width="1471" alt="Image" src="https://github.com/user-attachments/assets/7e922dcf-5cf3-40ec-94a4-f79128dd543d" />
